<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JokerWindow</class>
 <widget class="QMainWindow" name="JokerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Joker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_recent">
     <property name="title">
      <string>Open recent</string>
     </property>
     <addaction name="actionEmpty"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_recent"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Video"/>
    <addaction name="actionClose_video"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionProperties"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QMenu" name="menuVarial_Speed">
     <property name="title">
      <string>Varispeed</string>
     </property>
     <addaction name="action_3"/>
     <addaction name="action_1"/>
     <addaction name="action_0_5"/>
     <addaction name="action0"/>
     <addaction name="action0_5"/>
     <addaction name="action1"/>
     <addaction name="action3"/>
    </widget>
    <widget class="QMenu" name="menuGo_to">
     <property name="title">
      <string>Go to</string>
     </property>
     <addaction name="actionTimecode"/>
     <addaction name="actionNext_element"/>
     <addaction name="actionPrevious_element"/>
     <addaction name="actionNext_loop"/>
     <addaction name="actionPrevious_loop"/>
    </widget>
    <addaction name="actionPlay_pause"/>
    <addaction name="actionPlay_backward"/>
    <addaction name="actionStep_forward"/>
    <addaction name="actionStep_backward"/>
    <addaction name="actionStep_time_forward"/>
    <addaction name="actionStep_time_backward"/>
    <addaction name="menuVarial_Speed"/>
    <addaction name="actionChange_timestamp"/>
    <addaction name="menuGo_to"/>
    <addaction name="separator"/>
    <addaction name="actionSet_TC_in"/>
    <addaction name="actionSet_TC_out"/>
    <addaction name="actionLoop"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFullscreen"/>
    <addaction name="actionDisplay_the_control_panel"/>
    <addaction name="actionDisplay_the_information_panel"/>
    <addaction name="separator"/>
   </widget>
   <widget class="PhHelpMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSend_feedback"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actionForce_16_9_ratio"/>
    <addaction name="actionDeinterlace_video"/>
    <addaction name="actionUse_native_video_size"/>
    <addaction name="actionPicture_in_picture"/>
    <addaction name="actionSecond_screen"/>
   </widget>
   <widget class="QMenu" name="menuRythmo">
    <property name="title">
     <string>Rythmo</string>
    </property>
    <addaction name="actionSelect_character"/>
    <addaction name="actionHide_selected_peoples"/>
    <addaction name="separator"/>
    <addaction name="actionHide_the_rythmo"/>
    <addaction name="actionInvert_colors"/>
    <addaction name="actionDisplay_the_cuts"/>
    <addaction name="actionDisplay_the_vertical_scale"/>
    <addaction name="separator"/>
    <addaction name="actionDisplay_feet"/>
    <addaction name="actionSet_first_foot_timecode"/>
    <addaction name="actionSet_distance_between_two_feet"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControl"/>
   <addaction name="menuRythmo"/>
   <addaction name="menuVideo"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPlay_pause">
   <property name="text">
    <string>Play/pause</string>
   </property>
  </action>
  <action name="actionPlay_backward">
   <property name="text">
    <string>Play backward</string>
   </property>
  </action>
  <action name="actionStep_forward">
   <property name="text">
    <string>Step forward</string>
   </property>
  </action>
  <action name="actionStep_backward">
   <property name="text">
    <string>Step backward</string>
   </property>
  </action>
  <action name="actionStep_time_forward">
   <property name="text">
    <string>Step time forward</string>
   </property>
  </action>
  <action name="actionStep_time_backward">
   <property name="text">
    <string>Step time backward</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string notr="true">-3</string>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string notr="true">-1</string>
   </property>
  </action>
  <action name="action_0_5">
   <property name="text">
    <string notr="true">-0.5</string>
   </property>
  </action>
  <action name="action0">
   <property name="text">
    <string notr="true">0</string>
   </property>
  </action>
  <action name="action0_5">
   <property name="text">
    <string notr="true">0.5</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string notr="true">1</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string notr="true">3</string>
   </property>
  </action>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open video file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionChange_timestamp">
   <property name="text">
    <string>Change timestamp...</string>
   </property>
  </action>
  <action name="actionChange_font">
   <property name="text">
    <string>Change font...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties...</string>
   </property>
  </action>
  <action name="actionTest_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test mode</string>
   </property>
   <property name="toolTip">
    <string>Set Joker on Test mode for syncheck</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionTimecode">
   <property name="text">
    <string>Timecode...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionNext_text">
   <property name="text">
    <string>Next text</string>
   </property>
  </action>
  <action name="actionPrevious_text">
   <property name="text">
    <string>Previous text</string>
   </property>
  </action>
  <action name="actionNext_element">
   <property name="text">
    <string>Next element</string>
   </property>
  </action>
  <action name="actionPrevious_element">
   <property name="text">
    <string>Previous element</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSelect_character">
   <property name="text">
    <string>Select character...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionForce_16_9_ratio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Force 16/9 ratio</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionEmpty">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
  </action>
  <action name="actionInvert_colors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Invert colors</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClose_video">
   <property name="text">
    <string>Close video</string>
   </property>
  </action>
  <action name="actionSend_feedback">
   <property name="text">
    <string>Send feedback...</string>
   </property>
  </action>
  <action name="actionDeinterlace_video">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deinterlace video</string>
   </property>
  </action>
  <action name="actionHide_the_rythmo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide the rythmo</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionNext_loop">
   <property name="text">
	<string>Next loop</string>
   </property>
   <property name="shortcut">
	<string>Ctrl+Alt+Up</string>
   </property>
  </action>
  <action name="actionPrevious_loop">
   <property name="text">
	<string>Previous loop</string>
   </property>
   <property name="shortcut">
	<string>Ctrl+Alt+Down</string>
   </property>
  </action>
  <action name="actionDisplay_the_cuts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display the cuts</string>
   </property>
  </action>
  <action name="actionDisplay_the_vertical_scale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display the vertical scale</string>
   </property>
  </action>
  <action name="actionDisplay_the_information_panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display the information panel</string>
   </property>
  </action>
  <action name="actionDisplay_the_control_panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display the control panel</string>
   </property>
  </action>
  <action name="actionHide_selected_peoples">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide selected peoples</string>
   </property>
  </action>
  <action name="actionDisplay_feet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display feet</string>
   </property>
  </action>
  <action name="actionSet_first_foot_timecode">
   <property name="text">
    <string>Set first foot timecode...</string>
   </property>
  </action>
  <action name="actionSet_distance_between_two_feet">
   <property name="text">
    <string>Set distance between two feet...</string>
   </property>
  </action>
  <action name="actionUse_native_video_size">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use native video size</string>
   </property>
  </action>
  <action name="actionSet_TC_in">
   <property name="text">
    <string>Set TC in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionSet_TC_out">
   <property name="text">
    <string>Set TC out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionPicture_in_picture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Picture in picture</string>
   </property>
  </action>
  <action name="actionSecond_screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Second screen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhGraphicView</class>
   <extends>QWidget</extends>
   <header>PhGraphic/PhGraphicView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PhHelpMenu</class>
   <extends>QMenu</extends>
   <header>../../libs/PhCommonUI/PhHelpMenu.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
